lib/src/icons.dart

```diff
@@ -1,481 +1,422 @@
+import "dart:ui";
+
+import "package:flutter/foundation.dart";
+
 /// A icon element that can be used on aRdent smart glasses.
+@immutable
 class GYWIcon {
   /// The name of the icon
   final String name;
 
   /// The filename on the GYW device
   final String filename;
 
-  /// The width (in pixels) of the icon
-  final double width;
+  /// The size (in pixels) of the icon
+  final Size size;
+
+  /// The path of the associated PNG file in the assets folder
+  final String pathPng;
 
-  /// The height (in pixels) of the icon
-  final double height;
+  /// The path of the associated SVG file in the assets folder
+  final String pathSvg;
 
   /// Creates an icon.
   const GYWIcon({
     required this.name,
     required this.filename,
-    required this.width,
-    required this.height,
+    this.size = const Size(48, 48),
```

Comment:
> For an icon, size can be seen as the single value

Sure, we can make it a single double called size if you want.

> Moreover, Size usually targets Flutter widgets.

Color too usually targets Flutter widgets, so I don't see the problem. It's just an unopinionated utility class that can be used for anything.

So, I don't understand, what do you want me to do?

Reply:
Use height and width properties. There are a bunch of widgets working with these two instead of the `Size` object (`SizedBox`). I personally use it very rarely, except from the `MediaQuery.of(context).size` object. `Color` objects are way more common, are used everywhere in Flutter, and handled by IDE.

Reply:
Ok, I'm reverting the change.